### PR TITLE
Added documentation for using Orchard Core from a local NuGet packages

### PR DESCRIPTION
**Pull request by DrewBrasher**
Adding this documentation was suggested in this discussion: https://github.com/OrchardCMS/OrchardCore/discussions/10588#discussioncomment-2239153

### CODE REVIEW

File: src/docs/getting-started/local-package-source.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+* Update your nuget.config file so it points to your local feed.
+https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#packagesources
+```xml
+    <?xml version="1.0" encoding="utf-8"?>
+    <configuration>
+      <packageSources>
+        <clear />
+        <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+        <add key="MyFeed" value="\\{YourServer}\NuGetServer" />
```
Comment (Skrypt) [inline src/docs/getting-started/local-package-source.md:29]:
```suggestion
        <add key="MyFeed" value="\\{YourServer}\NuGetServer" />
        <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
```

[the remark this answers is not in the document]
Comment (DrewBrasher) [reply src/docs/getting-started/local-package-source.md:28]: updated in https://github.com/OrchardCMS/OrchardCore/pull/11284/commits/34d5ddff1e4b2d1939442ef4f1d67e56f499f54d

**Comment by ns8482e** [comment]
This is guide is optional for advance users and Adding this to getting started doesn’t seems to the right place as it’s confusing to new users

**Comment by Skrypt** [comment]
Move it to "How to guides". Else, we could add it to the repository Wiki.

**Comment by DrewBrasher** [comment]
What order should it go in the How to Guides left menu and should it also be listed in the Follow the Guides page?

**Comment by DrewBrasher** [comment]
Also, is there a better menu title than "Using local package source"?

**Comment by Skrypt** [comment]
Ok I changed my mind. Move this to "Key Topics" as "Using local NuGet packages" under "Using Docker".

**Review by Skrypt — APPROVED** [review]

**Comment by hishamco** [comment]
Why under Using Docker?

**Pull request merged**